cmd/promtool/main.go

```diff
@@ -20,8 +20,12 @@ import (
 	"path/filepath"
 	"strings"
 
+	yaml "github.com/ghodss/yaml"
```

Comment:
I believe the goal was to switch over to this library, as the semantics are much closer to the standard lib `encoding/json` package, where the `Marshal/Unmarshal` interface is a lot saner.

For the better or worse, the fact that it is already vendored is thanks to kubernetes/client-go.